plugins/osdn/ovs/controller.go

```diff
@@ -187,6 +213,10 @@ func (c *FlowController) Setup(localSubnetCIDR, clusterNetworkCIDR, servicesNetw
 	otx.AddPort(TUN, 2, "type=internal")
 	otx.AddPort(VOVSBR, 3)
 
+	// Table 253: rule version stamp; note action is hex bytes separated by '.'
+	pluginStamp := getPluginStamp(c.multitenant)
+	otx.AddFlow("%s, %s%s.%s", STAMP_TABLE, STAMP_ACTION, pluginStamp[0], pluginStamp[1])
+
```

Comment:
good idea. also, if this is going to get revised again, maybe change "STAMP" to "VERSION"?